R/PlatinumGenomes-QC/comparison/QC-Comparison.md

```diff
@@ -466,38 +469,38 @@ joinedResult <- inner_join(expectedResult, result, by=c("INDV"))
+  <tr> <td align="right"> 196.90 </td> <td align="right"> 194.35 </td> <td align="right"> -1.34 </td> <td align="right"> -1.47 </td> <td> NA12888 </td> <td align="right"> 280 </td> <td align="right"> 264 </td> <td align="right">  86 </td> <td align="right">  92 </td> </tr>
+  <tr> <td align="right"> 195.10 </td> <td align="right"> 194.57 </td> <td align="right"> -1.35 </td> <td align="right"> -1.43 </td> <td> NA12889 </td> <td align="right"> 275 </td> <td align="right"> 264 </td> <td align="right">  87 </td> <td align="right">  95 </td> </tr>
+  <tr> <td align="right"> 184.60 </td> <td align="right"> 195.24 </td> <td align="right"> 0.59 </td> <td align="right"> 0.58 </td> <td> NA12890 </td> <td align="right"> 253 </td> <td align="right"> 265 </td> <td align="right"> 225 </td> <td align="right"> 236 </td> </tr>
+  <tr> <td align="right"> 181.60 </td> <td align="right"> 191.70 </td> <td align="right"> 0.55 </td> <td align="right"> 0.60 </td> <td> NA12891 </td> <td align="right"> 250 </td> <td align="right"> 261 </td> <td align="right"> 219 </td> <td align="right"> 233 </td> </tr>
+  <tr> <td align="right"> 196.90 </td> <td align="right"> 193.42 </td> <td align="right"> -1.32 </td> <td align="right"> -1.50 </td> <td> NA12892 </td> <td align="right"> 282 </td> <td align="right"> 263 </td> <td align="right">  85 </td> <td align="right">  89 </td> </tr>
+  <tr> <td align="right"> 183.80 </td> <td align="right"> 193.58 </td> <td align="right"> 0.49 </td> <td align="right"> 0.52 </td> <td> NA12893 </td> <td align="right"> 252 </td> <td align="right"> 263 </td> <td align="right"> 217 </td> <td align="right"> 230 </td> </tr>
    </table>
 
-The logic in the query looks similar to vcftools [output_het method](http://sourceforge.net/p/vcftools/code/HEAD/tree/trunk/cpp/variant_file_output.cpp#l165) but there is clearly a difference.  TODO: investigate the difference further.
+The logic of the query is the same as vcftools [output_het method](http://sourceforge.net/p/vcftools/code/HEAD/tree/trunk/cpp/variant_file_output.cpp#l165), but the filtering criteria are slightly different, which explains the small difference in the results. The primary difference is due to using only SNPs in the query whereas vcftools uses both SNPs and indels. This is because the "expanded table" used in the query only contains non-variant segments for SNPs (i.e. indels matching the reference are not included), which results in inaccurate statistics for indels. There are a few other minor differences as well. For instance, vcftools discards the entire record if a [single sample is non-diploid](http://sourceforge.net/p/vcftools/code/HEAD/tree/trunk/cpp/entry_getters.cpp#l94) whereas the query only discards non-diploid samples within a record. Overall, the filtering differences are minor and the output of the query is largely consistent with vcftools.
```

Comment:
I think this paragraph should be broken up into three:

The logic of the query ...

The primary difference ...

There are a few other ...